lib/experimental/class-wp-theme-json-gutenberg.php

```diff
@@ -0,0 +1,29 @@
+<?php
```

Comment:
Any reason why this is not on 6.1 folder instead? Is it meant to stay experimental for longer?

Reply:
Moved. I kept this file so that we don't need to rename from _Gutenberg to _6_1 in the future.

Reply:
Well hopefully, at some point, we'd stop needing to update it :)

Reply:
I can't see that happening for quite a while